compiler/quilt/tools/build.py

```diff
@@ -293,21 +293,28 @@ def find(key, value):
                     for item in v:
                         for result in find(key, item):
                             yield result
+
+    try:
+        build_data = load_yaml(yaml_path)
```

Comment:
There are two calls to load_yaml in this try/except block that can throw exceptions for syntax errors. I think that means that the user won't be able to tell which file had the bad syntax.

If it's easier/cleaner, feel free to move the try/except into load_yaml (even though it's in core.py).